hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapPartitionContainer.java

```diff
@@ -50,8 +50,14 @@ public MultiMapContainer createNew(String name) {
     }
 
     public MultiMapContainer getOrCreateMultiMapContainer(String name) {
+        return getOrCreateMultiMapContainer(name, true);
+    }
+
+    public MultiMapContainer getOrCreateMultiMapContainer(String name, boolean isAccess) {
```

Comment:
We need to call this with `false` in the merge operation, so the `lastAccessTime` is not messed up